index.rst

```diff
@@ -922,38 +922,20 @@ dataset is “the persisted form of an in-memory object. It could be a single
 item, a composite, or a collection”). The relationship can occur as many
 datasets in one file, and can also occur as one dataset spread across many files.
 
-Many datasets in one file
-^^^^^^^^^^^^^^^^^^^^^^^^^
+Many (even potentially overlapping) datasets in one file is referred to as a "collection".
+Each dataset in a collection file is retrievable independently, provided an appropriate plugin exists to read it.
+Collections are typically written as a single "dataset-of-the-whole".
+Since there is nothing else special about them, they are not discussed further here.
```

Comment:
This section seems to beg the question of what the DataIds for the individual datasets vs. the collection as a whole are / can be.  But that does not need to be addressed to complete this PR.